File: 第一阶段/第一阶段上/面向对象/类和对象/Object03.java
public class Object03 {
	public static void main(String[] args){
		Person p1 = new Person();
		p1.age = 10;
		p1.name = "ะกร๗";

		Person p2 = p1;
		System.out.println(p2.age + p2.name);
	}
}

class Person{
	int age;
	String name;
}

File: 第一阶段/第一阶段上/面向对象/python/Object03.py
class Person:
	name = None
	age = None


p1 = Person()
p1.name = "小明"
p1.age = "18"

print("p1.name = " + p1.name + "\tp1.age = " + p1.age)


p2 = p1
print("p2.name = " + p2.name + "\tp2.age = " + p2.age)
